packages/@react-types/layout/src/grid.d.ts

```diff
@@ -13,7 +13,7 @@
 import {DOMProps, GridStyleProps} from '@react-types/shared';
 import {ReactNode} from 'react';
 
-export type Slots = {[key: string]: string};
+export type Slots = {[key: string]: any};
```

Comment:
unsure the better typing for this
it's really an object with anything in it, though usually with UNSAFE_className at least (but not required)